src/test/java/pages/MainPage.java

```diff
@@ -0,0 +1,29 @@
+package pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class MainPage extends BasePage {
+
+    @FindBy(className = "auth-bar__item--text")
+    public static WebElement entrance_Button;
+
+    @FindBy(xpath = "//div[@class='auth-form__label-part auth-form__label-part_1']/following::div[@class='auth-form__label-title']")
+    public static WebElement login_Via_NickName_Message;
```

Comment:
please use the same codename conventions as for all the variables in the project

Reply:
done